includes/admin_meeting.php

```diff
@@ -43,6 +43,8 @@
 
         $meeting = tsml_get_meeting();
 
+		$meeting_languages = isset($meeting->languages) && is_array($meeting->languages) ? $meeting->languages : []; // forces an array
```

Comment:
does this do anything? i don't see it being used below

also could you format this for PSR-12? 🙏